### ISSUE TITLE
Filter Syntax Error

### ISSUE THREAD

**Issue by soffes** [less/less.ruby#121]
If you try to add the following code to a selector, a parse error will result:

```
filter: progid:DXImageTransform.Microsoft.gradient(enabled='true', startColorstr=@top-color, endColorstr=@bottom-color, GradientType=1);
```

> on line 30: expected one of , ; } got ' after:

A lot of the `filter` values contain colons and other weird characters (thanks Microsoft). Escaping them would be acceptable, but it would obviously be preferred if Less just handled this correctly.

**Comment by jtousek** [comment]
yeah this would be very useful - the syntax %{ ...returned as it is... } maight be good enough for this
